File: ITManagementServer/src/main/java/models/responseModels/workers/WorkerGridResponseModel.java
package models.responseModels.workers;

import java.sql.Date;

public class WorkerGridResponseModel {
    private int WorkerId;
    private Integer DepartmentId;
    private String Name;
    private String Department;
    private double Salary;
    private Date HireDate;

    public int getWorkerId() {
        return WorkerId;
    }

    public void setWorkerId(int workerId) {
        WorkerId = workerId;
    }

    public Integer getDepartmentId() {
        return DepartmentId;
    }

    public void setDepartmentId(Integer departmentId) {
        DepartmentId = departmentId;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public String getDepartment() {
        return Department;
    }

    public void setDepartment(String department) {
        Department = department;
    }

    public double getSalary() {
        return Salary;
    }

    public void setSalary(double salary) {
        Salary = salary;
    }

    public Date getHireDate() {
        return HireDate;
    }

    public void setHireDate(Date hireDate) {
        HireDate = hireDate;
    }
}

File: ITManagementClient/Models/ResponseModels/Workers/WorkerGridResponseModel.cs
using System;

namespace ITManagementClient.Models.ResponseModels.Workers
{
    public class WorkerGridResponseModel
    {
        public int WorkerId { get; set; }

        public int DepartmentId { get; set; }

        public string Name { get; set; }

        public string Department { get; set; }

        public decimal Salary { get; set; }

        public DateTime HireDate { get; set; }
    }
}